src/particle/particle/convert.py

```diff
@@ -386,10 +401,10 @@ def produce_files(particle2008, particle2020, version, year):
     with open(particle2020, "w", newline="\n", encoding="utf-8") as f:
         f.write(version_header(particle2020, version))
         new_table.to_csv(f, float_format="%.12g")
-    f.close()
```

Comment:
This is automatic with a context manager. Even better, the context manager will still close the file even if you error out! (which is why we never manually call `.close()`)

Reply:
Good to know - did not know in case of error.